cypress/integration/group1/hostedWorkflows.ts

```diff
@@ -36,6 +36,8 @@ describe('Dockstore hosted workflows', () => {
     goToUnexpandedSidebarEntry('dockstore.org/A', /hosted/);
   }
 
+  const NEW_WORKFLOW_NAME = 'newHostedWorkflow';
```

Comment:
A very developer friendly name. Not that related to the bug, but while adding this could boundary push a bit and kill two birds with one stone. No spaces allowed, but could do "new_Hosted-Workflow8" for example

Reply:
Changed to a very not developer friendly name.